piperider_cli/runner.py

```diff
@@ -604,9 +605,9 @@ def get_dbt_state_dir(dbt_state_dir, dbt_config, ds, dbt_run_results):
     if not dbtutil.is_dbt_state_ready(dbt_state_dir):
         return None, f"[bold red]Error:[/bold red] No available 'manifest.json' under '{dbt_state_dir}'"
 
-    if not check_dbt_manifest_compatibility(ds, dbt_state_dir):
-        return None, f"[bold red]Error:[/bold red] Target mismatched. " \
-                     f"Please run 'dbt compile -t {dbt_config.get('target')}' to generate the new manifest"
+    if not os.environ.get('PIPERIDER_SKIP_TARGET_CHECK', None):
```

Comment:
found that `os.environ.get` returns a string
in this way, no matter `PIPERIDER_SKIP_TARGET_CHECK` is `'1'` or `'0'`, it would skip checking.
should we do a strict check like `os.environ.get('PIPERIDER_SKIP_TARGET_CHECK', 'false') ==  'true'`